docker-compose.yml

```
@@ -51,3 +51,20 @@ services:
       - env.d/development/crowdin
     user: "${DOCKER_USER:-1000}"
     working_dir: /app
+
+  docusaurus:
+    build: "./docs/"
+    ports:
+      - 3000:3000
+      - 35729:35729
+    user: "${DOCKER_USER:-1000}"
+    volumes:
+      - ./docs/docs:/app/docs
+      - ./docs/website/blog:/app/website/blog
+      - ./docs/website/core:/app/website/core
+      - ./docs/website/i18n:/app/website/i18n
+      - ./docs/website/pages:/app/website/pages
+      - ./docs/website/static:/app/website/static
+      - ./docs/website/sidebars.json:/app/website/sidebars.json
+      - ./docs/website/siteConfig.js:/app/website/siteConfig.js
```

Why not the following instead of all website-related volumes?

```suggestion
      - ./docs/website:/app/website
```

---

I presume because of the node_modules directory that was created when installing in the Dockerfile.

---

Does it help to install development dependencies globally (one single dependency at the time of writing) in the `docusaurus` image?

---

Should we cook/publish this image separately instead?